src/plugins/dashboard/public/dashboard_container/component/grid/_dashboard_grid.scss

```diff
@@ -41,8 +41,7 @@
  */
 .dshDashboardGrid__item--hidden {
   position: absolute;
-  top: -9999px;
-  left: -9999px;
```

Comment:
Out of curiosity, why the change from using `top`/`left` to `translate`?

Reply:
Switching from setting `top`/`left` to translate wasn't actually necessary. I think I was playing around with seeing if using translate vs top/left affected drawing time and timing of triggering the scroll, but I resolved this by using the `onTransitionEnd` listener on the panel ref. I'll revert this change.